app/models/podcast.rb

```diff
@@ -114,6 +116,10 @@ def feed_episodes
     feed
   end
 
+  def display_full_episode?(index)
```

Comment:
maybe should be a helper method instead? seems like a view concern?

Reply:
+1